app/models/user.rb

```diff
@@ -11,10 +9,43 @@ class User < ActiveRecord::Base
          :trackable,
          :validatable
 
+  after_create :send_admin_mail
+
   belongs_to :agency
 
-  scope :case_workers_by_agency, ->(agency = nil) { case_workers.where(agency: agency) }
-  scope :case_workers, -> { where(role: :case_worker) }
+  def self.case_workers_by_agency(agency = nil)
+    case_workers.where(agency: agency)
+  end
+
+  def self.case_workers
+    where(role: :case_worker)
+  end
+
+  def self.send_reset_password_instructions(attributes = {})
```

Comment:
I'm surprised they recommend that.  It seems much more verbose and error-prone.  Is that from a page on their wiki?